UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -133,7 +133,8 @@ public void CmdResist()
 			{
 				// Throw the player down to the floor for 15 seconds.
 				playerScript.registerTile.ServerStun(15);
-				SoundManager.PlayNetworkedAtPos("Bodyfall", transform.position, sourceObj: gameObject);
+				// JESTER
+				//SoundManager.PlayNetworkedAtPos("Bodyfall", transform.position, sourceObj: gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)